### PR TITLE
Add support for format 3

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

File: python/io.cpp [hunk 1]
```diff
@@ -12,8 +12,8 @@ PYBIND11_MODULE(_io, m)
 {
   py::class_<Header>(m, "_header")
     .def_readonly("images_in_block", &Header::imagesInBlock)
-    .def_readonly("rows", &Header::rows)
```
Comment (cryos) [inline python/io.cpp]: I don't think it matters lots, but I would assume the default is frame, and not prefix things referring to frames. I would instead prefix things referring to the larger thing, such as stemScanWidth, etc.

**Comment by cryos** [comment]
Looks reasonable to me, I think we can be reasonably relaxed about naming/API with a view to taking a pass when things settle a little more. Thanks for pushing on this Chris.

File: python/io.cpp (hunk 1 above)
Comment (cjh1) [reply python/io.cpp]: I just wanted to make it clear, as it was confusing to me, but we an iterate.

File: stempy/electron.cpp [hunk 2]
```diff
@@ -166,27 +169,51 @@ std::vector<uint32_t> maximalPoints(
...
+  }
+
+  // Raise an exception if we still don't have valid rows and columns
+  if (scanRows <= 0 || scanColumns <= 0) {
+    std::ostringstream msg;
+    msg << "No scan image size provided.";
+    throw std::invalid_argument(msg.str());
+  }
+
+  std::cout << scanRows << std::endl;
```
Comment (psavery) [inline stempy/electron.cpp]: Should we eliminate debug messages?

Comment (cryos) [reply stempy/electron.cpp]: I don't think it hurts having a few at this stage.

Comment (cjh1) [reply stempy/electron.cpp]: I will remove them, I didn't mean to leave them in!

**Pull request merged**